ethereum/datastructures/src/testFixtures/java/tech/pegasys/teku/datastructures/util/DataStructureUtil.java

```diff
@@ -126,6 +126,11 @@ public BLSSignature randomSignature() {
     return randomSSZList(classInfo, maxSize / 10, maxSize, valueGenerator);
   }
 
+  public <T> SSZList<T> randomSSZList(
+      Class<? extends T> classInfo, long maxSize, Supplier<T> valueGenerator, long numItems) {
+    return randomSSZList(classInfo, numItems, maxSize, valueGenerator);
+  }
```

Comment:
Why is this a new method just to reorder the params?

Reply:
Its to have configurable number of items in the sszlist. In the past we either had the `randomFullSSZList` option which filled the list to its maxSize, or the default `randomSSZList`, which would fill each list maxSize/10 number of items. 

In attester slashing, that would mean 0 items.